src/System.Security.Cryptography.Encryption.Aes/src/Internal/Cryptography/AesImplementation.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal sealed partial class AesImplementation : Aes
+    {
+        public sealed override ICryptoTransform CreateDecryptor(byte[] rgbKey, byte[] rgbIV)
+        {
+            return CreateTransform(rgbKey, rgbIV, encrypting: false);
+        }
+
+        public sealed override ICryptoTransform CreateEncryptor(byte[] rgbKey, byte[] rgbIV)
+        {
+            return CreateTransform(rgbKey, rgbIV, encrypting: true);
+        }
+
+        public sealed override void GenerateIV()
+        {
+            byte[] iv = new byte[BlockSize / BitsPerByte];
+            s_rng.GetBytes(iv);
+            IV = iv;
+        }
+
+        public sealed override void GenerateKey()
+        {
+            byte[] key = new byte[KeySize / BitsPerByte];
+            s_rng.GetBytes(key);
+            Key = key;
+        }
+
+        protected sealed override void Dispose(bool disposing)
+        {
+            base.Dispose(disposing);
+        }
+
+        private ICryptoTransform CreateTransform(byte[] rgbKey, byte[] rgbIV, bool encrypting)
+        {
+            if (rgbKey == null)
+                throw new ArgumentNullException("key");
+            int keySize = rgbKey.Length * BitsPerByte;
```

Comment:
Thanks for the feedback.  I've opened https://github.com/dotnet/corefx/issues/1981 for followup, but am going to merge as-is to best facilitate opening the library (given that this is an existing issue in the published package).